ghost/core/test/e2e-api/admin/pages.test.js

```diff
@@ -285,6 +285,57 @@ describe('Pages API', function () {
                 })
                 .expectStatus(200);
         });
+
+        describe('Access', function () {
+            describe('Visibility is set to tiers', function () {
+                it('Saves only paid tiers', async function () {
+                    const page = {
+                        title: 'Test Page',
+                        status: 'draft'
+                    };
+
+                    // @ts-ignore
+                    const products = await models.Product.findAll();
+
+                    const freeTier = products.models[0];
+                    const paidTier = products.models[1];
+
+                    const {body: pageBody} = await agent
+                        .post('/pages/', {
+                            headers: {
+                                'content-type': 'application/json'
+                            }
+                        })
+                        .body({pages: [page]})
+                        .expectStatus(201);
+
+                    const [pageResponse] = pageBody.pages;
+
+                    await agent
+                        .put(`/pages/${pageResponse.id}`)
+                        .body({
+                            pages: [{
+                                id: pageResponse.id,
+                                updated_at: pageResponse.updated_at,
+                                visibility: 'tiers',
+                                tiers: [freeTier, paidTier]
+                            }]
+                        })
+                        .expectStatus(200)
+                        .matchHeaderSnapshot({
+                            'content-version': anyContentVersion,
+                            etag: anyEtag,
+                            'x-cache-invalidate': anyString
+                        })
+                        .matchBodySnapshot({
+                            pages: [Object.assign({}, matchPageShallowIncludes, {
+                                published_at: null,
+                                tiers: [{id: paidTier.id, ...tierSnapshot}]
```

Comment:
Shouldn't this be 
```suggestion
                                tiers: [{id: paidTier.id}]
```
?

Reply:
This is to avoid timestamp issues with the snapshots. Otherwise I get failed tests because e.g. `updated_at` are not in sync

Reply:
This is to avoid timestamps issues with the `updated_at` fields are so on